### DISCUSSION TITLE
Case fans controlled by GPU temperature

### DISCUSSION THREAD

**Discussion by Merchal** [Rem0o/FanControl.Releases#455] [Q&A]
Hello,
Due to a failure in my RTX 2080 fan header, I am forced to run 2 x 120 mm standard case fans attached to my GPU with zip ties, which are connected to the motherboard via 4-pin connector. It is not pretty but it works.

They are currently running at constant speed the whole time but ideally, I would like these fans to run according to the GPU temperature. I have not been able to find a software capable of controlling fans connected to the motherboard and based on GPU temps.

Could you please let me know if FanControl can do that? And if so, would it be safe like what many people do with Afterburner or similar. That would be amazing!

Thank you.

**Comment by Rem0o** [comment]
I've seen a ton of people on Reddit doing exactly that, as long as the software recognize your mobo headers. Simply test it out.

**Reply by Merchal** [reply]
Thank you for your quick reply.

I had a look on Reddit and there are many people using, thanks for the help.
